src/components/List_Products.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+import { getItems, saveItem } from '../lib/api';
+
+function List_Products() {
```

Comment:
Same here. Please, review [this comment](https://github.com/the-collab-lab/tcl-39-smart-shopping-list/pull/16/files#r785042507).

Reply:
![image](https://user-images.githubusercontent.com/8689897/149564182-a25bc8f8-c607-4199-8ae2-411f05223da8.png)

Reply:
[Same here](https://github.com/the-collab-lab/tcl-39-smart-shopping-list/pull/16#discussion_r785191403).